.github/workflows/build-test.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   build:
+    if: (github.event_name == 'schedule' && github.repository == 'atmtools/atmtools-conda') || (github.event_name != 'schedule')
```

Comment:
Is there a neater syntax?

Reply:
As discussed on slack, this seems to be the only way for now.